src/cbc_sdk/platform/users.py

```diff
@@ -279,9 +497,79 @@ def __init__(self, doc_class, cb):
         super(UserQuery, self).__init__(None)
         self._doc_class = doc_class
         self._cb = cb
+        self._valid_emails = None
+        self._valid_userids = None
         self._count_valid = False
         self._total_results = 0
 
+    def email_addresses(self, addrs):
+        """
+        Limit the query to users with the specified E-mail addresses.  Call multiple times to add multiple addresses.
+
+        Args:
+            addrs (list): List of addresses to be added to the query.
+
+        Returns:
+            UserQuery: This object.
+        """
+        addr_set = set(addrs)
+        self._valid_emails = addr_set if self._valid_emails is None else addr_set.union(self._valid_emails)
+        return self
+
+    def email_address(self, addr):
```

Comment:
This adds _one_ E-mail address to the filter, as opposed to a _list_ of E-mail addresses. It's just syntactic sugar, really.

Reply:
The implementation makes sense, my concern is more on the complexity and number of functions. Would it make more sense to just keep this as the plural option and require them to submit it as a list instead of doing that internally?

Reply:
I don't know; would it? Chopping the singular methods won't save much, as they just call the plural methods after wrapping their argument in brackets.

Reply:
This reduces confusion as to which function to call. When I first saw them I did not notice that they were different

Reply:
Yes, they have different names. Do you still think the singular functions should be cut?

Reply:
It seems odd to me to have both, but that is my personal opinion so if you think there is a need for both then we should keep both

Reply:
I just decided to Marie Kondo those singular functions.  They don't really spark joy. :)